civitai-downloader-v2/src/ui/export.py

```
@@ -0,0 +1,881 @@
+            
+            for fmt in formats_to_include:
+                try:
+                    temp_options = ExportOptions(
+                        format=fmt,
+                        output_path=temp_path / f"data.{fmt.value}",
+                        scope=options.scope,
+                        filters=options.filters,
+                        include_metadata=options.include_metadata
+                    )
+                    
+                    result = await self.export_data(data, temp_options)
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)